winui/TreeGrid/Columns.md

```diff
@@ -307,13 +307,15 @@ You can set [TreeGridColumn.HeaderTemplate](https://help.syncfusion.com/cr/winui
 
 {% tabs %}
 {% highlight xaml %}
-<Page.Resources>
+<treeGrid:SfTreeGrid>
+<<treeGrid:SfTreeGrid.Resources>
```

Comment:
Open tag added twice. Modify and commit this.

Reply:
I have modified the tag